ldms/scripts/ldms-csv-export-sos

```diff
@@ -693,7 +698,7 @@ def parse_schema(fn):
         a = []
         for i in sch["attrs"]:
             if i["type"] in metric_types:
-            a.append(i["name"])
+            	a.append(["name"])
```

Comment:
@bschwal please verify that 4-space and not tab indents are used everywhere in this python script.
also, at 701 was converting i["name"] to ["name"] intentional? what does that fix?

Reply:
@baallan tab changes made accordingly, the "name" change was not intentional.

Reply:
assuming you've run it after the space changes, looks good to merge.

Reply:
I have with success.